src/dynapi/SDL_dynapi_procs.h

```diff
@@ -659,6 +659,7 @@ SDL_DYNAPI_PROC(void,SDL_MemoryBarrierReleaseFunction,(void),(),)
 SDL_DYNAPI_PROC(void,SDL_MemoryBarrierAcquireFunction,(void),(),)
 SDL_DYNAPI_PROC(SDL_JoystickID,SDL_JoystickGetDeviceInstanceID,(int a),(a),return)
 SDL_DYNAPI_PROC(size_t,SDL_utf8strlen,(const char *a),(a),return)
+SDL_DYNAPI_PROC(size_t,SDL_utf8strnlen,(const char *a, size_t b),(a,b),return)
```

Comment:
Do not insert new procs in dynapi, append to it. Running `gendynapi.pl` will do that for you.

Reply:
Oh, I see, my bad. I should have paid more attention. Thank you.
Is it alright now?

Reply:
From dynapi point of view, yes it's OK.